src/Native/Runtime/module.cpp

```diff
@@ -1270,3 +1270,49 @@ UInt32 GenericInstanceDesc::DacSize(TADDR addr)
     return GenericInstanceDesc::GetSize(dummyDesc.GetFlags(), arity);
 }
 #endif // DACCESS_COMPILE
+
+/* static */
+ModuleInfo * ModuleInfo::Create(void * pHeaderStart, void * pHeaderEnd)
```

Comment:
I'll try separating them. Our includes are in bad shape and it's hard to include just the headers we need without dragging a bunch of stuff to satisfy the compiler.